### DISCUSSION TITLE
Error on Status Code

### DISCUSSION THREAD

**Discussion by CalfCrusher** [projectdiscovery/httpx#866] [Q&A] [answered]
hello, sometimes httpx reports wrong status code. For example

`echo "https://ambassadors.shipt.com" |httpx -mc 200 -silent`

Returns a 200 status code

But

`curl -I https://ambassadors.shipt.com`

returns a 403 Forbidden

**Comment by ehsandeep** [comment, marked as answer]
@CalfCrusher you can easily debug the response with

```
echo "https://ambassadors.shipt.com" | httpx -debug
```

it's not a wrong response, instead, curl is blocked by that website for some reason.

**Comment by CalfCrusher** [comment]
@ehsandeep you right, my bad

**Reply by ehsandeep** [reply]
np
